lib/settings.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const templates = require('hof').template;
+const hoganExpressStrict = require('hogan-express-strict');
+const expressPartialTemplates = require('express-partial-templates');
+const bodyParser = require('body-parser');
+const path = require('path');
+
+module.exports = (app, config) => {
+
+  const viewEngine = config.viewEngine || 'html';
+
+  app.use((req, res, next) => {
+    req.baseUrl = config.siteroot + req.baseUrl;
+    res.locals.assetPath = config.assets;
+    res.locals.gaTagId = config.ga && config.ga.tagId;
+    next();
+  });
+  app.use((req, res, next) => {
+    if (req.baseUrl === 'undefined') {
```

Comment:
I haven't yet worked out why baseUrl is being set to 'undefined', but unfortunately it is.